README.md

```diff
@@ -115,3 +131,4 @@ shall be dual licensed as above, without any additional terms or conditions.
 [apache-url]: LICENSE-APACHE
 [actions-badge]: https://github.com/paradigmxyz/alphanet/workflows/unit/badge.svg
 [actions-url]: https://github.com/paradigmxyz/alphanet/actions?query=workflow%3ACI+branch%3Amain
+[eip3074-tools]: https://github.com/fgimenez/eip-3074-tools
```

Comment:
should we move this to either foundry or para org @gakonst @fgimenez ?

Reply:
> should we move this to either foundry or para org

:100: